### ISSUE TITLE
Cant make the scrip work

### ISSUE THREAD

**Issue by luchokoldo** [c64-ryan/theyarebillions-hotkeys#2]
Hi, idk if  something change in the game to block your scrip but I cant make it work in 2560x1440

edit: maybe is a issue with my 3 screen setup?

**Comment by ryuyan-dev** [comment]
Hey, thanks for trying to use my utility. I'm sorry it's not working out for you. Are you running it as an administrator?

Unfortunately I don't think I have a screen that I can use to test that resolution. I'll check.

**Comment by luchokoldo** [comment]
yes, I try it; now Im using this https://github.com/b-adkins/they_are_billions.ahk/pull/3/commits/3e4ded2b1810ecba2670272089dc5f2154ebd7b9 will be cool if you can implement the "take a screenshot and put the X/Y numbers" in your code

**Comment by ShadyPenguin** [comment]
Are you using the correct hotkeys? 

I'm running on the same resolution and I was having issues too which caused me to investigate. I had remapped my 'command center' hotkey to the tilda key which wasn't working with the script. 

The code has an internal memory that requires you to press specific buttons in order to use the hotkeys. I use the grid layout so I must press `enter` or `b` to then use the hotkeys provided by the script.

**Comment by ryuyandev** [comment]
Hey @luchokoldo, I know it's been years, but I was recently able to add back support for 1440p screens and have tested it, so if you ever want to play again, give it another try!
